.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,13 @@
-### Context
+## Please fill in the sections below
 
+After you submit your issue, the technical writing team from Central Digital and Data Office (CDDO) will discuss and prioritise it at our fortnightly triage meeting. We’ll then let you know if and when we’ll move it forward.
```

Comment:
Anything not in comments will appear in the submitted issue if not removed by the submitter, so to make things easier for them I think this part should be commented out:

```suggestion
<!--
## Please fill in the sections below

After you submit your issue, the technical writing team from Central Digital and Data Office (CDDO) will discuss and prioritise it at our fortnightly triage meeting. We’ll then let you know if and when we’ll move it forward.
-->
```